.gitignore

```diff
@@ -9,4 +9,3 @@ Thumbs.db
 # Node / npm
 .npm
 /node_modules/
-/package-lock.json
```

Comment:
Could we remove the package-lock file to avoid issues in the extensions that are JS related and people will not really care about? It also leads to merge conflicts often.

Reply:
removed.